services/core/MasterDriverAgent/master_driver/interfaces/bacnet.py

```diff
@@ -215,16 +232,21 @@ def parse_config(self, configDict):
 
         for regDef in configDict:
             #Skip lines that have no address yet.
-#             if not regDef['Point Name']:
-#                 continue
+            if not regDef['Volttron Point Name']:
```

Comment:
Should use regDef.get('Volttron Point Name') to avoid having a KeyError thrown if point doesn't exist.  Note all of these should have that same protection...